Core/MIPS/JitCommon/JitCommon.cpp

```diff
@@ -157,10 +159,12 @@ const char *ppsspp_resolver(struct ud*,
 		*offset = addr - (uint64_t)MIPSComp::jit->GetBasePtr();
 		return "jitcode";
 	}
+#if defined(_M_IX86) || defined(_M_X64) || defined(MIPS)
```

Comment:
Okay.

We used to build on powerpc but the jit was ripped out because we weren't maintaining it properly.  Anything big endian will have issues until someone puts some time into it again, unfortunately, although a decent chunk of parts have support.

-[Unknown]